pkg/model/postgresql_aws_periodic_backup.go

```diff
@@ -0,0 +1,125 @@
+package model
+
+import (
+	"github.com/keycloak/keycloak-operator/pkg/apis/keycloak/v1alpha1"
+	v13 "k8s.io/api/batch/v1"
+	"k8s.io/api/batch/v1beta1"
+	v1 "k8s.io/api/core/v1"
+	v12 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func PostgresqlAWSPeriodicBackup(cr *v1alpha1.KeycloakBackup) *v1beta1.CronJob {
+	return &v1beta1.CronJob{
+		ObjectMeta: v12.ObjectMeta{
+			Name:      cr.Name,
+			Namespace: cr.Namespace,
+			Labels: map[string]string{
+				"app":       ApplicationName,
+				"component": PostgresqlBackupComponent,
+			},
+		},
+		Spec: v1beta1.CronJobSpec{
+			Schedule: cr.Spec.AWS.Schedule,
+			JobTemplate: v1beta1.JobTemplateSpec{
```

Comment:
This looks to be a duplicate of the postgresql_aws_backup. It would be nice if it could be reused rather than duplicated.

Reply:
Good point. I abstracted Containers[], which is the biggest part of the payload. Looks much better now.